integration/experiment/common_args.py

```diff
@@ -34,61 +34,94 @@
 Common command line arguments for experiments.
 '''
 
-
-def add_output_dir(parser):
-    parser.add_argument('-o', '--output-dir', dest='output_dir',
-                        action='store', default='.',
+def setup_run_args(parser):
+    """Add common arguments for all run scripts:
+       --output-dir --node-count --trial-count --cool-off-time
+    """
+    add_output_dir(parser)
+    add_node_count(parser)
+    add_trial_count(parser)
+    add_cool_off_time(parser)
+
+
+def add_output_dir(parser, default='.'):
```

Comment:
Enabling override of defaults upon call to add_*() may not be all that useful since we can always call parser.set_defaults() after the fact.  Maybe we should not make that change.